index.html

```diff
@@ -575,8 +575,9 @@ <h3>Use Cases and Requirements</h3>
 
         <p>
 This specification includes both required and optional fields. An <a>issuer</a>
-MUST include all required fields. A <a>verifier</a> SHOULD <a>validate</a> all
-required fields. Optional fields MAY be ignored by <a>issuers</a> and
+MUST include all required fields. A <a>verifier</a> SHOULD check that all
+required fields satisfy the normative requirements for each.
```

Comment:
```suggestion
MUST include all required fields. A <a>verifier</a> SHOULD check that each
required field satisfies the normative requirements for that field.
```

Reply:
This could be marked resolved, as it was handled in https://github.com/w3c/vc-data-model/pull/1256/commits/57402c686583a490a4c4f33fc214bfa2efce509f